docs/source/basics/environment/docker_recipes.md

```diff
@@ -9,3 +9,9 @@ RUN curl -L https://sourceforge.net/projects/bowtie-bio/files/bowtie2/2.4.4/bowt
     unzip bowtie2-2.4.4.zip &&\
     mv bowtie2-2.4.4-linux-x86_64 bowtie2
 ```
+
+## Run Docker in Docker
+
+Latch provides a base image which includes Docker. Specify the `--base-image docker` argument to `latch init` to use the image. Latch provides a comprehensive example of running a containerized `bowtie2` aligner in a Latch workflow, which can be found by running `latch init --template docker ...`.
+
+Use [`--base-image docker` with `latch init`](../../subcommands.md##latch-init) to use a base workflow environment which includes Docker. An example of running a containerized `bowtie2` aligner in a Latch workflow can be found using `latch init --template docker my_bowtie2_example`.
```

Comment:
this is redundant given the previous paragraph, del

Reply:
O fuk I forgot to delete the previous one